server/column_scope.cc

```diff
@@ -215,7 +215,8 @@ runIncrementalT(const std::vector<BoundSqlExpression> & exprs,
                     
                 size_t numRows = endOffset - startOffset;
 
-                Val results[exprs.size()];
```

Comment:
Placement new does call the constructor, and you need to call the destructor manually.  std::uninitialized_fill basically calls placement new over a range.